com.github.sdbg.debug.ui/plugin.xml

```diff
@@ -1077,6 +1079,15 @@
             <bundle name="com.github.sdbg.debug.core"/>
       </component>
    </extension>
+   
+   <!-- Used to launch chrome from another plugin like GWT -->
+   <extension
+         point="com.github.sdbg.debug.ui.launchChrome">
+      <extensions
+            class="com.github.sdbg.ui.launchers.ChromeLaunchShortcutExt">
+      </extensions>
+   </extension>
+
```

Comment:
I'm probably missing the obvious here, but why do we even need to design an Eclipse extension here?
My understanding is that Eclipse extensions are only necessary when plugin "special" contributes to plugin "generic" a piece of code (say, an action, a command or suchlike) by implementing a generic interface defined ***in plugin "generic"***.  This way, plugin "generic" can then call into plugin 
"special"'s action/command, without even knowing that plugin "special" exists in the first place.

In our case, both the interface (old IChromeLaunchShortcutExt / new IChromeLaunch) and the implementation (old ChromeLaunchShortcutExt / new ChromeLaunchShortcut) are defined in the same plugin - SDBG. Since the GWT plugin now anyway depends on SDBG (as it needs to know about the IChromeLaunchShortcutExt/IChromeLaunch interface), I don't think it makes sense to introduce any extension point. You can just as well just call from the GWT plugin the ChromeLaunch class (or a factory which hides it and exposes only the interface).

If we really want to do it with extensions, then I think it should be the other way around:
- The GWT plugin should define the IChromeLaunch interface (appropriately renamed to IDebugLaunch or suchlike)
- The GWT plugin should define an extension point so that debuggers like SDBG can plug there and provide implementations of IDebugLaunch
- The GWT plugin should no longer depend on SDBG
- SDBG should (optionally) depend on the GWT plugin and provide one or more implementations of IDebugLaunch